packages/react/src/components/FileUploader/index.js

```diff
@@ -2,36 +2,37 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-import Button from '../Button';
 import Icon from '../Icon';
 import Spinner from '../Spinner';
-import Text from '../Text';
+import UiText from '../UiText';
 import UploaderItem from '../UploaderItem';
 
-import { ICONS, INTENTS, refShapes } from '../../constants';
+import { refShapes } from '../../constants';
 import { safeInvoke } from '../../utils';
 
 const defaultTranslate = ({ defaultText }) => defaultText;
 
 const texts = {
   add_document: {
-    id: 'flamingo-file-uploader-add-document',
+    id: 'flamingo.file-uploader.add-document',
```

Comment:
Because with `-` you're forced to have every locale at the same level
```
{
  "flamingo-stuff-a": "foo",
  "flamingo-stuff-b": "bar",
  "app-related-heading": "baz"
}
```

Where with `.` you can nest locales
```
{
  "flamingo": {
    "stuff-a": "foo",
    "stuff-b": "bar",
  },
  "app-related-heading": "baz"
}
```

It adds a hierarchy layer, and you can fold/unfold the whole content into your editor